web/src/lib/domain-logic/sidebarCardsHelpers.ts

```diff
@@ -45,7 +45,7 @@ export const getVisibleCertifications = (
   });
 };
 export const sortFundingsForUser = (fundings: Funding[], userData?: UserData): Funding[] => {
```

Comment:
[pebble] We should either update the tests for this function or create a new one to account for the new behavior introduced by calling `sortFundingsForUser`.